docs/index.md

```diff
@@ -1391,13 +1391,13 @@ The "nyan" reporter is exactly what you might expect:
 
 ### TAP
 
-The TAP reporter emits lines for a [Test-Anything-Protocol][] consumer.
+The "TAP" reporter emits lines for a [Test-Anything-Protocol][] consumer.
 
 ![test anything protocol](images/reporter-tap.png?withoutEnlargement&resize=920,9999){:class="screenshot" lazyload="on"}
 
 ### Landing Strip
 
-The Landing Strip (`landing`) reporter is a gimmicky test reporter simulating a plane landing :) unicode ftw
+The Landing Strip ("landing") reporter is a gimmicky test reporter simulating a plane landing :) unicode ftw
```

Comment:
Isn't it better to show them as code block rather than just quoted?

Reply:
I think either is fine, but consistency would be best.  I went with quotes because all the rest were using quotes. 

Personally, I think it might be better to explicitly indicate that the quoted/code-block is the string name used when choosing the reporter.  Maybe something like: 

```text
reporter name = "landing"
```

Or maybe to keep changes minimal, the use the inline code-block with quotes?   
Eg. The Landing Strip (`"landing"`) reporter...  
Or is that overkill?  I kind of like the quotes because it draws extra attention and implies it must be used exactly as written. (But, I guess the code block does as well.)

I'll change it to what ever the majority/senior contributors prefer!  (So since you seem pretty senior, I'll change them all to code blocks (or whatever) if you prefer!)

Reply:
For reporter's formatting discussion please see:
https://github.com/mochajs/mocha/issues/4054

Reply:
please use double quotes "landing" for consistency reasons.